### ISSUE TITLE
Possible real-world application of your findings

### ISSUE THREAD

**Issue by yundai424** [uchicago-computation-workshop/ed_awh#27]
Thank you for your presentation!
I am wondering if there are some possible real-world application of your results, for example, psychotropic medicine to cure ADHD or bipolar disorder, or even application to headset game devices that enhance the immersive gaming experience?

Thank you！

**Comment by edawh** [comment]
Yes, it is possible that an approach like this could be useful for brain-machine interfaces. The responses might be a bit slow at present for really effective real-time tracking though, so I suspect further development would be needed.
